### ISSUE TITLE
thread panics when adding new group

### ISSUE THREAD

**Issue by AFriemann** [Rukenshia/saml2aws-auto#59]
**Describe the bug**

cli panics when trying to add a new group.

**To Reproduce**

1. configure saml2aws-auto
2. run saml2aws-auto groups add ...

**Expected behavior**

Successfully adds a group.

**Logs**

```
$ RUST_BACKTRACE=1 saml2aws-auto --skip-password-manager groups add default --prefix "myorg-" --role MyRole
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', src/groups.rs:36:21
stack backtrace:
   0: backtrace::backtrace::libunwind::trace
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.44/src/backtrace/libunwind.rs:86
   1: backtrace::backtrace::trace_unsynchronized
             at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.44/src/backtrace/mod.rs:66
   2: std::sys_common::backtrace::_print_fmt
             at src/libstd/sys_common/backtrace.rs:78
   3: <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt
             at src/libstd/sys_common/backtrace.rs:59
   4: core::fmt::write
             at src/libcore/fmt/mod.rs:1063
   5: std::io::Write::write_fmt
             at src/libstd/io/mod.rs:1426
   6: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:62
   7: std::sys_common::backtrace::print
             at src/libstd/sys_common/backtrace.rs:49
   8: std::panicking::default_hook::{{closure}}
             at src/libstd/panicking.rs:204
   9: std::panicking::default_hook
             at src/libstd/panicking.rs:224
  10: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:470
  11: rust_begin_unwind
             at src/libstd/panicking.rs:378
  12: core::panicking::panic_fmt
             at src/libcore/panicking.rs:85
  13: core::panicking::panic
             at src/libcore/panicking.rs:52
  14: saml2aws_auto::groups::command
  15: saml2aws_auto::main
  16: std::rt::lang_start::{{closure}}
  17: main
  18: __libc_start_main
             at ../csu/libc-start.c:308
  19: <unknown>
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
[1]    5810 exit 101   RUST_BACKTRACE=1 saml2aws-auto --skip-password-manager groups add default
```

**Desktop (please complete the following information):**
 - OS: Void Linux (kernel 5.4.39_1)

**Comment by AFriemann** [comment]
Could be an issue with musl?

**Comment by Rukenshia** [comment]
Hi! Thanks for the report.

Thanks, the issue will now output correct error messaging. If you use the `--skip-password-manager` flag, you need to supply a password through the `--password` flag.

This is fixed with 1.8.1.
